README.md

```diff
@@ -65,11 +65,16 @@ stack test
 ### Git Branches
 
 If you're a GHC developer and want to update Haddock to work with your
-changes, you should be working on `ghc-head` branch instead of `master`.
+changes, you should be working on `ghc-head` branch.
 See instructions at
 https://ghc.haskell.org/trac/ghc/wiki/WorkingConventions/Git/Submodules
 for an example workflow.
 
-The `master` branch usually requires a GHC from the latest GHC stable
-branch. The required GHC version can be inferred from the version
-bounds on `ghc` in the respective `.cabal` files.
+### Updating `html-test`
+
+When accepting any changes in the output of `html-test`, it is important
+to use the `--haddock-path` option. For example:
+
+```
+cabal new-run -- html-test --haddock-path $(find dist-newstyle/ -executable -type f -name haddock) --accept
+```
```

Comment:
Not sure if there's a more elegant way to accept changes?!

Reply:
In theory, if  `html-test` had a `build-tools: haddock` declaration; `new-run` should put the intra-package `haddock` exe on `$PATH` for the process invocation of `exe:html-test`.

Reply:
@hvr Won't `build-tool-depends` only affect what things are on `$PATH` at build time? If it is also supposed to affect `$PATH` in `new-run`, that's cool but [undocumented](http://cabal.readthedocs.io/en/latest/developing-packages.html?highlight=build-tool-depends#pkg-field-build-tool-depends).